README.md

```diff
@@ -137,5 +139,39 @@ const producer = await createProducer(config, (err, report) => {
 ...
 ```
 
+## GCP Environments with Service Accounts
+
+For Python, you can initialize your client as follows:
+```
+...
+
+from kafka.oauth.abstract import AbstractTokenProvider
+
+class MyTokenProvider(AbstractTokenProvider):
+
+    #include kafka_gcp_credentials_server.py content without the local server parts and build_message() method.  
```

Comment:
I do not like this, I'd prefer if you break [kafka_gcp_credentials_server.py](https://github.com/googleapis/managedkafka/blob/main/kafka-auth-local-server/kafka_gcp_credentials_server.py) in two modules so the logic to get_kafka_access_token is encapsulated in a module that one can include here.